### ISSUE TITLE
Set-HPOVInitialPassword

### ISSUE THREAD

**Issue by tlauder** [HewlettPackard/POSH-HPOneView#319]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
Password to change

### Actual Behavior
Error in CMDLET

### Steps to reproduce
```PowerShell
Set-HPOVInitialPassword -Appliance "x.x.x.x" -OldPassword "admin" -NewPassword "NewPasswordHere" -Verbose
VERBOSE: [SET-HPOVINITIALPASSWORD] Bound PS Parameters:
Key         Value
---         -----
Appliance   x.x.x.x
OldPassword admin
NewPassword NewPasswordHere
Verbose     True


VERBOSE: [SET-HPOVINITIALPASSWORD] Called from: <ScriptBlock>
VERBOSE: [SET-HPOVINITIALPASSWORD] Appliance Session not found. Running FTS sequence?
VERBOSE: [SET-HPOVINITIALPASSWORD] Creating temporary Session object
Error formatting a string: Input string was not in a correct format..
At C:\Program Files\WindowsPowerShell\Modules\HPOneView.400\HPOneView.400.psm1:88546 char:14
+                 "[{0}] $($Appliance | fl * | out-string)" -f $MyInvoc ...
+                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: ([{0}]
Conne...es : {}
:String) [], RuntimeException
    + FullyQualifiedErrorId : FormatError

VERBOSE: [SEND-HPOVREQUEST] BEGIN
VERBOSE: [SEND-HPOVREQUEST] Bound PS Parameters:
Key      Value
---      -----
Hostname HPOneView.Appliance.Connection
uri      /rest/users/changePassword
method   POST
body     @{userName=Administrator; oldPassword=admin; newPassword=NewPasswordHere}


VERBOSE: [SEND-HPOVREQUEST] Process
VERBOSE: [SEND-HPOVREQUEST] Hostname value:
ConnectionID Name UserName AuthLoginDomain Default
------------ ---- -------- --------------- -------
99                                         False


VERBOSE: [SEND-HPOVREQUEST] Processing '' appliance connection request. 1 of 1
VERBOSE: [SEND-HPOVREQUEST] Requested URI '/rest/users/changePassword' to ''
VERBOSE: [SEND-HPOVREQUEST] We have reached the URI Whitelist condition block. Unauth request allowed for'/rest/users/changePassword'.
Set-HPOVInitialPassword : Cannot validate argument on parameter 'Appliance'. The argument is null or empty. Provide an
argument that is not null or empty, and then try the command again.
At line:1 char:1
+ Set-HPOVInitialPassword -Appliance "x.x.x.x" -OldPassword "admin ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Set-HPOVInitialPassword], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Set-HPOVInitialPassword
```

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion` or `$PSLibraryVersion`): 4.0.1587.3400
HPE OneView Appliance Version (`Get-HPOVVersion -ApplianceVer`):  4.00.07-0330056
Output from `$PSVersionTable` on your Windows Host:
$PSLibraryVersion

LibraryVersion Path
-------------- ----
4.0.1587.3400  C:\Program Files\WindowsPowerShell\Modules\HPOneView.400

# Output from $PSVersionTable
```PowerShell
$PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.14393.2068
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.2068
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by ChrisLynchHPE** [comment]
Thank you for reporting this.  I was about to publish an update over the weekend, and during final regression testing, I came across this one and another defect that I'm trying to resolve before an updated library is published.

**Comment by tlauder** [comment]
No problem. Does the other defect have to do with the Set-HPOVApplianceDateTime command? If not, I can open another issue. The exact command from the help file throws an error: Set-HPOVApplianceDateTime -NTPServers 10.55.1.1,10.54.1.1 -PollingInterval 60. If this is not the defect you are working on, I will open another issue.

**Comment by ChrisLynchHPE** [comment]
Yes, and even with Set-HPOVApplianceNetworkConfig.

**Comment by tlauder** [comment]
OK, cool. thank you for the response!

**Comment by ChrisLynchHPE** [comment]
This was fixed in the v4.0.1612.2800 release.  But since then, I have also released a new version [Release 4.0.1630.2612](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v4.0.1630.2612) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.400/4.0.1630.2612).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If this issue persists, we can re-open this request.
